### ISSUE TITLE
Migration issue on Jormungandr (v2020-03-16 => v2020-04-01 | 07)

### ISSUE THREAD

**Issue by piotr-iohk** [cardano-foundation/cardano-wallet#1552]
# Context
 
After migration `v2020-03-16` => `v2020-04-01` (or `v2020-04-07`) one cannot send tx from Shelley wallet or migrate Byron wallet... 

Note: as a workaround one can remove and restore the wallet again.

| Information  | -                                                     |
| ---          | ---                                                   |
| Version      | cardano-wallet  `v2020-03-16` => `v2020-04-01`, jormungandr 0.8.15  |
| Platform     | Probably all, discovered on Linux          |
| Installation | from release          |

# Steps to Reproduce

1. Launch server using cardano-wallet `v2020-03-16`
```
cd lib/jormungandr/test/data/jormungandr/test_scripts
./wallet_launch.sh
```
2. Create a wallet (e.g. faucet shelley wallet from integration tests: `absurd seat ball together donate bulk sustain loop convince capital peanut mutual notice improve jewel`)
3. Stop wallet and start wallet  `v2020-04-01` or `v2020-04-07`
4. Submit transaction from that wallet

## Expected behavior

TX is successful.

## Actual behavior
 
```
Code = 403,
Json = {"code":"no_root_key","message":"I couldn't find a root private key for the given wallet: 6e262e86ac68ab5f71d0b6dfe27cdc0609db5198. However, this operation requires that I do have such a key. Either there's no such wallet, or I don't fully own it."}
```

---

# Resolution 

---

# QA

**Comment by Anviking** [comment]
`cardano-wallet-byron` is not affected?

**Comment by piotr-iohk** [comment]
No, there is no `v2020-03-16` for cardano-wallet-byron. Migration `v2020-04-01` => `v2020-04-07` seems to be fine.

**Comment by piotr-iohk** [comment]
fixed with #1717.
